Python/syntax_test_python.py

```diff
@@ -1350,6 +1350,74 @@ class Starship:
 #        ^ punctuation.separator.annotation.variable.python
 #                                   ^ keyword.operator.assignment
 
+# Type comments - type: ignore must be by itself.
+
+primes = 5  # type: ignore # type: not-a-type-comment
+#           ^ comment.line.type-hint punctuation.definition.comment.python comment.line.number-sign.python
+#             ^^^^ comment.line.type-hint keyword.other.annotation.type-comment
+#                 ^ comment.line.type-hint punctuation.separator.annotation.type-comment
+#                   ^^^^^^ comment.line.type-hint keyword.other.annotation.type-comment.ignore
+#                          ^ comment.line.type-hint comment.line.number-sign.python punctuation.definition.comment.python
+#                            ^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.type-hint comment.line.number-sign.python
```

Comment:
From this test, but also below, it looks like you're not popping.

Reply:
Yeah, the comment scopes are stacked, and multiple times so. While this certainly isn't clean, I doubt it would break anything, so I didn't comment on it.